packages/cli/src/index.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 // 3p
+import chalk from 'chalk';
```

Comment:
The CLI already uses the [colors](https://www.npmjs.com/package/colors) library to highlight outputs. Can update the import?
`import { red, yellow } from 'colors/safe';`

Reply:
👍